### DISCUSSION TITLE
SX1276 low range

### DISCUSSION THREAD

**Discussion by novvere** [Lora-net/LoRaMac-node#1332] [Q&A] [answered]
I'm porting LoRaMAC-node to MAX32670 MCU using a rfm95w board using EU868 region.
The JoinRequest is always  received by the gateway when both gateway and node are in the same room but the joinAccept is never detected by the node and, if I move the gateway to the next room the node messages do not reach the gateway.

The gateway shows bad values for RSSI around -100
![image](https://user-images.githubusercontent.com/7149270/179833761-e2cbae6b-e20c-47ee-871e-966e61e427f1.png)

I have tried with 2 different gateways, different types of antenna and different breakout board for the rfm95w module with the same results.

Has anybody used this stack with RFM95W module?
Any idea what can be wrong?

Thanks,
Miguel

**Comment by mluis1** [comment, marked as answer]
If I am not mistaken the RFM95W radio is the same as SX1276 radio.

This radio has 2 RF possible outputs HF and LF.

The first thing to check is to which RF output is your antenna connected to. Then you need to check if your ported files are using the right RF output.

All this depends on your final PCB schematic.

At the following link you may find some clues on how to solve the observed behavior.
https://www.thethingsnetwork.org/forum/t/loramac-node-not-sending-nucleol476rg-dragino-1-4/25273/13?u=mluis1

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

**Reply by novvere** [reply]
You are wright, as you say in the link, in RFM95W module the antenna is tied to the PA_BOOST pin.
I don't have the module schematics to check but using your solution RSSI goes from -100 to -29.

I have just replaced the contents of 
`static uint8_t SX1276GetPaSelect( int8_t power )`

by 
```
static uint8_t SX1276GetPaSelect( int8_t power )
{
    return RF_PACONFIG_PASELECT_PABOOST;
}
```
